modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -1096,4 +1096,29 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-171627. Verify that that on Target Market, just 1 ~ 3 options can be selected "
+	@priority = 4
+	test ViewTargetMarketOptionsError {
+		property solutions.acceptance = "true";
+		property test.name.skip.portal.instance = "PRMMDFRequest#OnlyManagersCanSeeStateManager";
+
+		task ("Given I go to add a new MDF Request") {
+			PRMMarketing.gotoAddNewMDF();
+		}
+
+		task ("When the user selects 4 different Target Market options") {
+			PRMMDFRequest.fillGoalsPage(targetMarketOptions = "Education,Energy,Automotive,Agriculture");
+		}
+
+		task ("Then the error message that exceeded the limit should appear") {
+			PRMMDFRequest.verifyErrorMessege(errorMessage = "You have exceed the choose limit");
```

Comment:
Hi @pedroamarall, there is a typo in your macro name. Replace `Messege` with `Message`, please! Also, I think that it would be better if we replace the word `verify` with `view`, so your macro name will be `viewErrorMessage`. What do you think?

Reply:
Done, thanks!